ansible/server-state-playbooks/www/tests/check_redirects.py

```diff
@@ -29,11 +29,18 @@
 
     ('/site/products', '/products'),
     ('/site/products/bio-formats', '/bio-formats'),
+    ('/site/products/bio-formats/downloads', '/bio-formats/downloads/'),
     ('/site/products/omero', '/omero'),
+    ('/site/products/omero/downloads', '/omero/downloads/'),
     ('/site/products/omero/feature-list', '/omero/new'),
+    ('/site/products/omero/secvuln', '/security/'),
```

Comment:
It occurs to me that we've dropped a prefix here. I think that's ok, but we'll have to keep that in mind when naming future secvulns.